test/ui/agent-view-ui.test.ts

```
@@ -0,0 +1,537 @@
+		agentViewUI = new AgentViewUI(app, plugin);
+
+		// Create mock container
+		container = document.createElement('div');
+		document.body.appendChild(container);
+
+		// Helper to create element mock
+		const createElMock = jest.fn().mockImplementation((tag: string, options?: any) => {
+			const el = document.createElement(tag);
+			if (options?.cls) el.className = options.cls;
+			if (options?.attr) {
+				Object.entries(options.attr).forEach(([k, v]) => el.setAttribute(k, v as string));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)